firmware/common2015/utils/logger/logger.hpp

```diff
@@ -61,6 +64,24 @@ extern bool isLogging;
 extern uint8_t rjLogLevel;
 
 /**
+ * [Collects the stream log message into a single string to print]
+ * @param logLevel [The "importance level" of the called log message.]
+ * @param source   [The source of the message.]
+ * @param format   [The string format for displaying the log message.]
+ */
+class LogHelper : public std::stringstream {
+public:
+    LogHelper(uint8_t logLevel, const char* source, int line, const char* func);
+    ~LogHelper();
+
+private:
+    uint8_t m_logLevel;
```

Comment:
Let's drop the leading `m` to keep naming consistency with the rest of our code.